composer.json

```diff
@@ -22,7 +22,10 @@
         "phpunit/phpunit": "^4.7"
     },
     "autoload": {
-        "psr-0": { "WindowsAzure\\": "" }
+        "psr-0": { "WindowsAzure\\": "WindowsAzure/" }
```

Comment:
This is the problem that the tests would not run. This will cause WindowsAzure to be loaded from Current dir/WidowsAzure/WindowsAzure.